BinaryTree.py

```diff
@@ -0,0 +1,33 @@
+class TreeNode(object):
+    """TreeNode is a unit which is used in my implementation of Binary Tree."""
+
+    def __init__(self, value=None, left=None, right=None):
+        self.__value = value
+        self.__left = left
+        self.__right = right
+
+    def get_value(self):
+        return self.__value
+
+    def get_left(self):
+        return self.__left
+
+    def get_right(self):
+        return self.__right
+
+    def set_left(self, child):
+        self.__left = child
+
+    def set_right(self, child):
+        self.__right = child
+
+
+class BinaryTree(object):
```

Comment:
You can actually drop this class if the rest of your methods are able to work directly on a TreeNode and they know that `None` corresponds to an empty tree.